mockup/patterns/structure/pattern-structureupdater.js

```diff
@@ -0,0 +1,40 @@
+/* Structure Updater pattern.
+ *
+ * Options:
+ *    titleSelector(string): CSS selector to match title elements for the current context ('').
+ *    descriptionSelector(string): CSS selector to match description elements of the current context to be updated ('').
+ *
+ * Documentation:
+ *    The Structure Updater pattern updates the information of the current context after a user navigated to a new folder.
+ *
+ */
+
+define([
+  'jquery',
+  'pat-base'
+], function($, Base) {
+  'use strict';
+
+  var StructureUpdater = Base.extend({
+    name: 'structureupdater',
+    trigger: '.template-folder_contents',
```

Comment:
please note the trigger. it's what's in the body class, set by https://github.com/plone/plone.app.layout/blob/master/plone/app/layout/globals/layout.py#L187
That way, I do not need to update this method in plone.app.layout to add a ``pat-structureupdater`` class or so.
I think it's fine, even if it doesn't follow closely the naming conventions.